pkgs/os-specific/linux/systemd/0019-tpm2_context_init-fix-driver-name-checking.patch

```diff
@@ -0,0 +1,28 @@
+From 4f5a1c2c7409dee9a8e81f75141a4fd9dd889af7 Mon Sep 17 00:00:00 2001
+From: Nick Cao <nickcao@nichi.co>
+Date: Sun, 15 Jan 2023 20:15:55 +0800
+Subject: [PATCH] tpm2_context_init: fix driver name checking
+
+"libtss2-tcti-" is patched into "/nix/store/xxxxxxxxx-tpm2-tss-3.2.0/lib/libtss2-tcti-",
+resulting in the concated filename being an absolute path, tripping
+checks in filename_is_valid
```

Comment:
Can you add some description here, that the string in L18 of the patch gets patched to an absolute path, or maybe even do part of the transformation in that patch, to make this less confusing?